src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -148,6 +148,12 @@ internal sealed partial class ConsoleHost
 
             try
             {
+                // We need to read the settings file before we create the console host
+                string[] tempArgs = new string[args.GetLength(0)];
+                args.CopyTo(tempArgs, 0);
+
+                CommandLineParameterParser.EarlyParse(tempArgs);
```

Comment:
I've created issue https://github.com/PowerShell/PowerShell/issues/7460 to move this call to UnmanagedPSEntry.Start prior to PSEtwLog.LogConsoleStartup().

Reply:
Thanks